privacyidea/lib/apps.py

```diff
@@ -43,32 +43,31 @@
 """
```

Comment:
since You are already working on `lib/apps.py` could You also check if the `test_lib_apps.py` tests run? I think they should work out of the box now.